modules/API/external/external_api.js

```diff
@@ -130,25 +150,11 @@ function JitsiMeetExternalAPI(domain, room_name, width, height, parentNode,
 
     this.url += "#jitsi_meet_external_api_id=" + id;
 
-    var key;
-    if (configOverwrite) {
-        for (key in configOverwrite) {
-            if (!configOverwrite.hasOwnProperty(key) ||
-                typeof key !== 'string')
-                continue;
-            this.url += "&config." + key + "=" + configOverwrite[key];
-        }
-    }
-
-    if (interfaceConfigOverwrite) {
-        for (key in interfaceConfigOverwrite) {
-            if (!interfaceConfigOverwrite.hasOwnProperty(key) ||
-                typeof key !== 'string')
-                continue;
-            this.url += "&interfaceConfig." + key + "=" +
-                interfaceConfigOverwrite[key];
-        }
-    }
+    this.url += "&config."
```

Comment:
Add some ifs, if the object is undefined or empty we end up with a weird URL ending in `&config.`